tests/models/test_implicit_als.py

```diff
@@ -105,6 +105,38 @@ def test_basic(
             actual,
         )
 
+    @pytest.mark.parametrize("fit_features_together", (False, True))
+    def test_consistent_with_pure_implicit(self, dataset: Dataset, fit_features_together: bool, use_gpu: bool) -> None:
+        base_model = AlternatingLeastSquares(factors=10, num_threads=2, iterations=30, use_gpu=use_gpu)
+        users = np.array([10, 20, 30, 40])
+        self._init_model_factors_inplace(base_model, dataset)
+
+        model_for_wrap = deepcopy(base_model)
+        wrapped_model = ImplicitALSWrapperModel(model=model_for_wrap, fit_features_together=fit_features_together)
+        wrapped_model.fit(dataset)
+        actual_reco = wrapped_model.recommend(
+            users=users,
+            dataset=dataset,
+            k=3,
+            filter_viewed=False,
+        )
+
+        ui_csr = dataset.get_user_item_matrix(include_weights=True)
+        base_model.fit(ui_csr)
+        for user_id in users:
```

Comment:
Isn't it correct that implicit can recommend for multiple users? Why do you need the cycle?

Reply:
I believe it's just simpler to match recos from different methods this way